.gitignore

```diff
@@ -1,4 +1,6 @@
 _esy
+_build
```

Comment:
Can you help me understand how this folder was created? Were you using `esy dune ...` for something?

Reply:
Yep! I've been looking for type errors with `esy dune build --watch @check @runtest`

Reply:
I think you'll need this in the `devDependencies`

```json
    "esy-fswatch": "ulrikstrid/esy-fswatch:esy.json#7328e82b62c1691cd13623fffde0b66d0a0f9498",
```

Reply:
It is safe, `esy b` is the build environment for production, same as used when releasing a package, `esy` is the build environment + current env, which is used for development. 

Also that doesn't create the `_build` probably it was something else.

Reply:
`esy b` is meant for development too. command environment is not reproducible and running Dune in it is not recommended. `DUNE_BUILD_DIR` isn't set in cmd environment and explains the _build folder

Reply:
> esy sh -c 'echo $DUNE_BUILD_DIR'
It's empty. Checkout `esy command-env` too

I don't understand why you think development environment should be any less reproducible then in the release mode.

Reply:
> It's empty

It's not, if it is it's a bug, commad_env is a strict superset of the build environment.

![DUNE_BUILD_DIR showing up on both places](https://user-images.githubusercontent.com/3393115/136435437-106784bd-2c87-4c65-a9ae-2edcf679ba6a.png)

> I don't understand why you think development environment should be any less reproducible then in the release mode.

Because development is hacking land and being reproducible is not desirable at all when you want to try things. Again